docs/Configuring-Pantheon/Network-vs-Node.md

```diff
@@ -0,0 +1,12 @@
+description: Configuring Pantheon at the network level compared to the node level 
+<!--- END of page meta data -->
+
+# Network vs Node Configuration 
+
+Pantheon is configured at the network level and the node level. 
+
+Network wide settings are specified in the genesis file.  For example, `difficulty` for Proof of Work 
```

Comment:
The consensus algo would be a nice example too IMO.

Reply:
Fixed